data/json/items/ammo/40.json

```diff
@@ -44,10 +44,10 @@
     "ammo_type": "40",
     "casing": "40_casing",
     "range": 14,
-    "damage": 28,
-    "pierce": 2,
+    "damage": 26,
```

Comment:
Will do but separate PR. I don't want to delay a large body of work over a trivial difference.